backend/alembic/versions/800f48024ae9_add_id_to_connectorcredentialpair.py

```diff
@@ -0,0 +1,60 @@
+"""Add ID to ConnectorCredentialPair
+
+Revision ID: 800f48024ae9
+Revises: 767f1c2a00eb
+Create Date: 2023-09-19 16:13:42.299715
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.schema import Sequence, CreateSequence
+
+# revision identifiers, used by Alembic.
+revision = "800f48024ae9"
+down_revision = "767f1c2a00eb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    sequence = Sequence("connector_credential_pair_id_seq")
+    op.execute(CreateSequence(sequence))  # type: ignore
+    op.add_column(
+        "connector_credential_pair",
+        sa.Column(
+            "id", sa.Integer(), nullable=True, server_default=sequence.next_value()
+        ),
+    )
+    op.add_column(
+        "connector_credential_pair",
+        sa.Column("name", sa.String(), nullable=True),
+    )
+
+    # fill in IDs for existing rows
+    op.execute(
+        "UPDATE connector_credential_pair SET id = nextval('connector_credential_pair_id_seq') WHERE id IS NULL"
```

Comment:
How could this not be null?

Reply:
If the migration is run twice for some reason